website/docs/docs/build/metrics.md

```diff
@@ -324,6 +345,66 @@ Note that `value` must be defined as a string in YAML, because it will be compil
         value: "'2020-01-01'"
 ```
 
+### Calendar
+The dbt_metrics package contains a [basic calendar table](https://github.com/dbt-labs/dbt_metrics/blob/main/models/dbt_metrics_default_calendar.sql) that is created as part of your `dbt run`. It contains dates between 2010-01-01 and 2029-12-31. 
+
+If you want to use a custom calendar, you can replace the default with any table which meets the following requirements:
+- Contains a `date_day` column. 
+- Contains the following columns: `date_week`, `date_month`, `date_quarter`, `date_year`, or equivalents. 
+- Additional date columns need to be prefixed with `date_`, e.g. `date_4_5_4_month` for a 4-5-4 retail calendar date set. Dimensions can have any name (see [dimensions on calendar tables](#dimensions-on-calendar-tables)).
+
+To do this, set the value of the `dbt_metrics_calendar_model` variable in your `dbt_project.yml` file: 
+```yaml
+#dbt_project.yml
+config-version: 2
+[...]
+vars:
+    dbt_metrics_calendar_model: my_custom_calendar
+```
+
+<VersionBlock firstVersion="1.3">
+
+### Configuration 
```

Comment:
suggesting a tab item for this to reduce scrolling. i took the liberty of writing it out for you, thoughts?
```

<!--tabs for config and project.yml -->

<Tab>
<TabItem value="config" label="Metric yml">

```yml
# in metrics.yml
version: 2
metrics:
  - name: config_metric
    label: Example Metric with Config
    model: ref('my_model')
    calculation_method: count
    timestamp: date_field
    time_grains: [day, week, month]
    config:
      enabled: True
```

</TabItem>

<TabItem value="project" label="dbt_project.yml">

```yml
# in dbt_project.yml
metrics: 
  your_project_name: 
    +enabled: true
```

</TabItem>
</Tabs>

<!--End of tabs for config and project.yml -->
```

Reply:
```
<!--tabs for config and project.yml -->

<Tabs>
<TabItem value="config" label="Metric yml">

```yml
# in metrics.yml
version: 2
metrics:
  - name: config_metric
    label: Example Metric with Config
    model: ref('my_model')
    calculation_method: count
    timestamp: date_field
    time_grains: [day, week, month]
    config:
      enabled: True
```

`</TabItem>`

```
<TabItem value="project" label="dbt_project.yml">

```yml
# in dbt_project.yml
metrics: 
  your_project_name: 
    +enabled: true
```

```
</TabItem>
</Tabs>
```

`<!--End of tabs for config and project.yml -->`
```
```

Reply:
```
<!--tabs for config and project.yml -->

<Tab>
<TabItem value=“config” label=“Metric yml”>
```

```
yml
# in metrics.yml
version: 2
metrics:
  - name: config_metric
    label: Example Metric with Config
    model: ref(‘my_model’)
    calculation_method: count
    timestamp: date_field
    time_grains: [day, week, month]
    config:
      enabled: True
```


```
</TabItem>

<TabItem value=“project” label=“dbt_project.yml”>
```

```
yml
# in dbt_project.yml
metrics: 
  your_project_name: 
    +enabled: true

```

```
</TabItem>
</Tabs>

<!--End of tabs for config and project.yml -->
```

Reply:
Love it! Implementing this